src/js/tech/flash.js

```diff
@@ -69,6 +70,15 @@ class Flash extends Tech {
   createEl() {
     let options = this.options_;
 
+    // If video.js is hosted locally you should also set the location
+    // for the hosted swf, which should be relative to the page (not video.js)
+    // Otherwise this adds a CDN url.
+    // The CDN also auto-adds a swf URL for that specific version.
+    if (!options.swf) {
+      let protocol = ('https:' === window.location.protocol ? 'https://' : 'http://');
+      options.swf = protocol+'vjs.zencdn.net/swf/'+pkg.dependencies['videojs-swf']+'/video-js.swf';
```

Comment:
I'd rather be more explicit with compiled dependencies. I made a note on the [cdn repo](https://github.com/videojs/cdn) "Patch releases should never change, including hosted dependencies". I don't want any risk of breaking patch-specific versions on the CDN.